src/components/PlantsV2/components/PlantingSiteProgressCard.tsx

```diff
@@ -0,0 +1,46 @@
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import { Box, Typography, useTheme } from '@mui/material';
+import strings from 'src/strings';
+import React, { useMemo } from 'react';
+import { useAppSelector } from 'src/redux/store';
+import { selectPlantingSite } from 'src/redux/features/tracking/trackingSelectors';
+
+type PlantingSiteProgressCardProps = {
+  plantingSiteId?: number;
```

Comment:
make this mandatory since this component is specific to a selected planting site?

Reply:
Yeah that would make sense